.github/workflows/1_create_release_pr.yml

```
@@ -16,42 +16,42 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 10
     steps:
-
-    - name: Sanitise workflow inputs # Should be 1st step
-      uses: cylc/release-actions/stage-1/sanitize-inputs@v1
-
-    - name: Checkout repo
-      uses: actions/checkout@v2
-      with:
-        ref: ${{ env.BASE_REF }}
-
-    - name: Check CONTRIBUTING.md
-      uses: cylc/release-actions/check-shortlog@v1
-
-    - name: Setup Python
-      uses: actions/setup-python@v2
-      with:
-        python-version: '3.x'
-
-    - name: Create & checkout PR branch
-      uses: cylc/release-actions/stage-1/checkout-pr-branch@v1
-
-    - name: Set the package version
-      uses: cylc/release-actions/stage-1/set-python-package-version@v1
-      with:
-        init-file: 'cylc/uiserver/__init__.py'
-        pypi-package-name: 'cylc-uiserver'
-
-    - name: Update "released on" date in changelog
-      continue-on-error: true
-      uses: cylc/release-actions/stage-1/update-changelog-release-date@v1
-      with:
-        changelog-file: 'CHANGES.md'
-
-    - name: Test build
-      uses: cylc/release-actions/build-python-package@v1
-
-    - name: Create pull request
-      uses: cylc/release-actions/stage-1/create-release-pr@v1
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Sanitise workflow inputs # Should be 1st step
+        uses: cylc/release-actions/stage-1/sanitize-inputs@v1
+
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ env.BASE_REF }}
+          fetch-depth: 0  # need to fetch all commits to check contributors
+
+      - name: Check CONTRIBUTING.md
+        uses: cylc/release-actions/check-shortlog@v1
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+
+      - name: Create & checkout PR branch
+        uses: cylc/release-actions/stage-1/checkout-pr-branch@v1
+
+      - name: Set the package version
+        uses: cylc/release-actions/stage-1/set-python-package-version@v1
+        with:
+          init-file: 'cylc/uiserver/__init__.py'
+          pypi-package-name: 'cylc-uiserver'
+
+      - name: Update "released on" date in changelog
+        continue-on-error: true
+        uses: cylc/release-actions/stage-1/update-changelog-release-date@v1
+        with:
+          changelog-file: 'CHANGES.md'
+
+      - name: Test build
+        uses: cylc/release-actions/build-python-package@v1
+
+      - name: Create pull request
+        uses: cylc/release-actions/stage-1/create-release-pr@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Would be nice to have the tests status badge
```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        with:
          test-workflows: test.yml
```